pkgs/os-specific/linux/wpa_supplicant/default.nix

```diff
@@ -48,6 +48,7 @@ stdenv.mkDerivation rec {
     CONFIG_HS20=y
     CONFIG_HT_OVERRIDES=y
     CONFIG_IEEE80211AC=y
+    CONFIG_IEEE80211AX=y
```

Comment:
This is unneeded, it is already enabled by default [wpa_supplicant](https://w1.fi/cgit/hostap/tree/wpa_supplicant/defconfig#n487)

Reply:
Not in 2.10 https://w1.fi/cgit/hostap/tree/wpa_supplicant/defconfig?h=hostap_2_10